### PR TITLE
Project page

### PR DESCRIPTION
**Pull request by EthanJStark**
Hi Prrr!

To run, please visit '/trossello' route. Thanks!

### CODE REVIEW

File: src/Button.js [hunk 1]
```diff
@@ -5,15 +5,14 @@ export default class Button extends Component {
 
   constructor( props ) {
     super( props )
-    this.state = {
```
Comment (deadlyicon) [inline src/Button.js]: if you remove this you can remove the whole constructor

File: src/Button.js [hunk 2]
```diff
@@ -5,15 +5,14 @@ export default class Button extends Component {
...
-      label: 'Contact'
-    }
   }
 
   render() {
     return (
       <div className='button grey'>
-        {this.state.label}
+        <a href={this.props.url}>
+        {this.props.label}
```
Comment (deadlyicon) [inline src/Button.js]: indent

File: src/ProjectImage.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class ProjectImage extends Component {
```
Comment (deadlyicon) [inline src/ProjectImage.js]:
```js
export default class ProjectImage extends Component {
```

File: src/Tag.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import './Tag.css'
+
+export default class Tag extends Component {
+
+  render(){
+    return (
+    <div className='tag-box'>
```
Comment (deadlyicon) [inline src/Tag.js]: indent?

Comment (deadlyicon) [inline src/Tag.js]: You have a component named `Tag` but the className of the outer most component is `tag-box`. Why not make the outer `div` classname the same as the component. `<div className="Tag">` and then give the internal elements a scoped classname `<span className="tag-thing">`?

File: src/TextBox.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  border: 1px solid #e9e9e9;
+  margin: .5em auto;
+  height: auto;
+  width: 200px;
+  text-align: center;
+  line-height: 2em;
+  color: #7e7e7e;
+}
+
+.badge-link {
```
Comment (deadlyicon) [inline src/TextBox.css]: kill  this?

File: src/TextBox.css [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+img {
```
Comment (deadlyicon) [inline src/TextBox.css]: this is kind of crazy. Consider moving this to a classname

File: src/TextBox.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+export default class TextBox extends Component {
+
+  constructor( props ) {
+    super( props )
+  }
+
+  render() {
+    return (
+      <div className='textbox-wrapper'>
```
Comment (deadlyicon) [inline src/TextBox.js]: same recommendation here RE: the classname matching the component name

File: src/TextBox.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import './TextBox.css'
+
+export default class TextBox extends Component {
+
+  constructor( props ) {
```
Comment (deadlyicon) [inline src/TextBox.js]: this constructor is not needed

File: src/TextBox.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import './TextBox.css'
+
+export default class TextBox extends Component {
+
```
Comment (deadlyicon) [inline src/TextBox.js]: consider defining `propTypes`

File: src/project.css [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.project-title {
+  color: #7e7e7e;
+  text-align: center;
+  font-weight: 100;
+  margin: .2em;
+}
+
+.description {
```
Comment (deadlyicon) [inline src/project.css]: this is a very generic classname. consider changing it to `.project-description` ?

File: src/routes.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Router, Route } from 'react-router'
+
+import App from './App'
+import Trossello from './Trossello'
+
+const Routes = ( props ) => {
+    return <Router { ...props }>
```
Comment (deadlyicon) [inline src/routes.js]: formatting. indented too far

File: src/TextBox.css (hunk 6 above)
Comment (shakalee14) [inline src/TextBox.css]: ...

**Pull request merged**